### DISCUSSION TITLE
Intel Arc A770 support

### DISCUSSION THREAD

**Discussion by Envious-Data** [Rem0o/FanControl.Releases#1295] [General]
Hello

Ive recently bought an Arc A770 and I find the drivers for my install dont have fan control so I was curious if I could help with support for FanControl via testing, logging or what ever else you'd want me to grab or do from my PC assuming I know how.

- Adam

**Comment by Rem0o** [comment]
Is there a known API to get fan control on these? I wouldn't think so.

**Reply by Envious-Data** [reply]
since it was breifly in the driver panel I would assume it might be possible to hijack that, but Ive not been able to re-enable the fan control.
